packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -334,3 +384,34 @@ export class LocalRuntimeController extends RuntimeController {
 		this.emit("reloadComplete", data);
 	}
 }
+
+/**
+ * @returns Container options suitable for building or pulling images,
+ * with image tag set to well-known dev format.
```

Comment:
"well-known dev format` -> @see <I think there is a fn computing that>

is what we call image tag throughout this PR  `name:tag`? is this the right Docker terminology?